sw/device/silicon_creator/lib/crypto/tests/rsa_3072_verify_set_testvectors.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import os
+import sys
+
+import hjson
+from mako.template import Template
+
+'''
+Read in a JSON test vector file, convert the test vector to C constants, and
+generate a header file with these test vectors.
+'''
+
+# Number of 32-bit words in a 3072-bit number
+RSA_3072_NUMWORDS = int(3072 / 32)
+
+# Template file name
+TEMPLATE = 'rsa_3072_verify_testvectors.h.tpl'
+
+# Default output file name
+DEFAULT_OUTFILE = 'rsa_3072_verify_testvectors.h'
+
+
+def rsa_3072_int_to_hexwords(x):
+    '''Convert a 3072-bit integer to a list of 32-bit integers (little-endian).'''
+    out = []
+    for _ in range(RSA_3072_NUMWORDS):
+        out.append(x & ((1 << 32) - 1))
+        x >>= 32
+    return ['{:#010x}'.format(w) for w in out]
```

Comment:
minor: you may want to add a check here just in case:
```
if x != 0:
  raise OverflowError()
```

Reply:
This raised an interesting issue: I tried to implement the check and it was actually triggered on the Wycheproof test set, which apparently contains at least one signature that's simply too big to fit in 3072 bytes. Since our types rule out accepting this entire input, I think that for now it makes sense to just truncate it. However, I'll add a comment to explain!